pkg/scheduler/metrics/metric_recorder.go

```diff
@@ -17,7 +17,9 @@ limitations under the License.
 package metrics
 
 import (
-	"k8s.io/component-base/metrics"
+	"time"
+
+	k8smetrics "k8s.io/component-base/metrics"
```

Comment:
/hold
can we use metrics directly here?

Reply:
Good catch. Fixed.